src/auth.ts

```diff
@@ -59,6 +59,7 @@ export async function redirectToLobby(
 
   const exchangeDid = await did.exchange()
   const writeDid = await did.write()
+  const sharedRepo = !!document.body.querySelector("iframe[src$=\"/ipfs.html\"]") && typeof SharedWorker === "function"
```

Comment:
I feel like *eventually* we want to have some kind of webnative class/instance/state.
Since the code to initialize the shared worker is in webnative's control, it should be possible to keep track of that as state, instead of relying on this.

But that'd be a bigger refactor, just throwing that over the fence :smile:

Reply:
I agree that this feels a little fragile. Perhaps as an interim step we could put an `id` or something on the iframe so at least we have a little more confidence we're checking the right one.

Reply:
Good point, added 👌